### DISCUSSION TITLE
osu!lazer - Skin Editor Gui Bug

### DISCUSSION THREAD

**Discussion by plantasaur** [ppy/osu#15065] [Q&A] [answered]
So when I open the skin editor gui, save the configuration and try to leave the map, my game doesn't return to the menu, it just shows the background of the map and after a few seconds it crashes. Here is the video of me demonstrating the problem. https://www.youtube.com/watch?v=TSbzVVAh1iY

**Comment by bdach** [comment, marked as answer]
please attach game logs.

**Reply by plantasaur** [reply]
Where are the logs saved?

**Reply by bdach** [reply]
* `%AppData%/osu/logs` (on Windows),
* `~/.local/share/osu/logs` (on Linux and macOS),
* `Android/Data/sh.ppy.osulazer/logs` (on Android).

**Reply by plantasaur** [reply]
[logs.zip](https://github.com/ppy/osu/files/7332941/logs.zip)

**Reply by bdach** [reply]
duplicate of https://github.com/ppy/osu/discussions/14850 / https://github.com/ppy/osu/discussions/14884.

the database system is in the middle of a rewrite right now, so i'll abstain from converting this to an issue as this will likely be fixed during the transition away from EF to realm.
